content/learn/book/getting-started/setup/_index.md

```diff
@@ -200,6 +200,9 @@ Bevy can be built just fine using default configuration on stable Rust. However
 
     For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
 
+* **Cranelift Codegen**: This uses a new nightly-only codegen that can be 300% faster than LLVM! However it only currently works on Linux and is generally still immature.
+    [Cranelift codegen setup](https://bjorn3.github.io/2023/10/31/progress-report-oct-2023.html).
```

Comment:
Maybe link to the setup section of the readme instead? Unlike this page it will be kept up to date if the setup changes.

Reply:
Also it is possible to compile all dependencies with the LLVM backend and optimizations, but the main game crate with Cranelift to get much better runtime performance.

Reply:
@bjorn3 I've tried that before and ran into issues where I would get random segfaults.

Reply:
There were some abi incompatibilities in the past, but most should be fixed now.